lib/init.php

```diff
@@ -57,3 +57,58 @@ function gutenberg_menu() {
 	);
 }
 add_action( 'admin_menu', 'gutenberg_menu', 9 );
+
+function disable_scripts() {
+	// disable loading and enqueuing block editor scripts and styles
+	// can't be `__return_false` because this function is already registered for this filter
+	return false;
+}
+
+add_filter( 'should_load_block_editor_scripts_and_styles', 'disable_scripts' );
+
+// Add `editorScript` to Gutenberg block.json data if it's missing
+function add_core_editor_script( $metadata ) {
+	if ( ! empty( $metadata['name'] ) && str_starts_with( $metadata['name'], 'core/' ) ) {
+		$block_name = str_replace( 'core/', '', $metadata['name'] );
+		if ( ! isset( $metadata['editorScript'] ) ) {
+			if ( isset( $metadata['file'] ) && strstr( $metadata['file'], 'gutenberg') ) {
```

Comment:
Nit: maybe these should be combined in a single `if` or we could invert the top one to avoid deep nesting.